src/main/java/com/gtnewhorizons/angelica/models/MixinModels.java

```diff
@@ -0,0 +1,26 @@
+package com.gtnewhorizons.angelica.models;
+
+import com.gtnewhorizons.angelica.compat.nd.Quad;
+import com.gtnewhorizons.angelica.models.json.JsonModel;
+import com.gtnewhorizons.angelica.models.json.Loader;
+import com.gtnewhorizons.angelica.models.json.Variant;
+import it.unimi.dsi.fastutil.objects.ObjectImmutableList;
+import net.minecraft.util.ResourceLocation;
+
+import java.util.List;
+
+public class MixinModels {
```

Comment:
I'd like a name for this that doesn't start with `Mixin` since normally that naming convention is used for mixin classes. `VanillaModels` perhaps.